source/slang/slang-ir-autodiff-fwd.cpp

```diff
@@ -1690,8 +1690,6 @@ SlangResult ForwardDiffTranscriber::prepareFuncForForwardDiff(IRFunc* func)
 {
     insertTempVarForMutableParams(autoDiffSharedContext->moduleInst->getModule(), func);
     removeLinkageDecorations(func);
-
-    performPreAutoDiffForceInlining(func);
```

Comment:
Maybe an alternative way to approach this issue is that instead of moving around the inlining pass, 
we just allowed `IRForwardDifferentiate` (and other derivative insts) to appear in the func body during transcription (and treat them like a regular function modifier like how we treat `IRSpecialize`)

Since we now support insts like  `IRCall(IRBackwardDifferentiate(IRForwardDifferentiate(fn)))`, this should be fine? It will just get handled in the next sweep.